src/components/formgithub/FormGithub.tsx

```diff
@@ -0,0 +1,72 @@
+import { FormGithubProps } from '@/interfaces/ui/form-github-props.interface'
+import React, { useState } from 'react'
+
+export const FormGithub: React.FC<FormGithubProps> = ({
```

Comment:
Cambiar el nombre del componente a GithubForm
```suggestion
export const GithubForm: React.FC<FormGithubProps> = ({
```